docs/README.md

```diff
@@ -461,6 +461,54 @@ vmselect requests stats via [/api/v1/status/tsdb](#tsdb-stats) API from each vms
 This may lead to inflated values when samples for the same time series are spread across multiple vmstorage nodes
 due to [replication](#replication) or [rerouting](https://docs.victoriametrics.com/cluster-victoriametrics/?highlight=re-routes#cluster-availability).
 
+### Track ingested metrics usage
```

Comment:
It doesn't mention which HTTP API calls contirbute to the stats. Could you please add this to the docs?

Reply:
It's good point actually. Currently, it will track requests for export ( if reduce_mem_usage=false) and federate API calls. It'll only mislead users. 

 Unfortunately, it requires to add additional option to search request, which will affect vmselect-> vmstorage communication. new version of vmselect cannot be used with older version of vmstorage. However, old version of vmselect can be used with new version of vmstorage. It must provide a zero downtime migration path - at first update storage nodes, after that update select nodes.